cli/package.json

```diff
@@ -13,7 +13,7 @@
     "@ledgerhq/hw-transport-node-ble": "5.6.0",
     "@ledgerhq/hw-transport-node-hid": "5.6.0",
     "@ledgerhq/ledger-core": "^5.3.0",
-    "@ledgerhq/live-common": "^11.1.1",
+    "@ledgerhq/live-common": "file:.yalc/@ledgerhq/live-common",
```

Comment:
this should not be committed. 
in fact you no olnger need to do `yalc publish` or `yalc add` . it's only for developers now.